### PR TITLE
update CLI support bundle command

### PR DESCRIPTION
**Pull request by miaawong**
#### What this PR does / why we need it: This PR updates all CLI support bundle commands in admin console. 

Screenshot 2023-05-24 at 9 40 22 AM

#### Which issue(s) this PR fixes:

Fixes # https://app.shortcut.com/replicated/story/69757/show-customer-improved-cli-support-bundle-command-for-collecting-support-bundles

#### Does this PR introduce a user-facing change?

```release-note
Update improved CLI support-bundle commands
```

### CODE REVIEW

File: web/src/components/UploadAirgapBundle.jsx [hunk 1]
```diff
@@ -313,37 +314,9 @@ class UploadAirgapBundle extends React.Component {
     }, 1000);
   };
 
-  getSupportBundleCommand = async (slug) => {
-    const res = await fetch(
-      `${process.env.API_ENDPOINT}/troubleshoot/app/${slug}/supportbundlecommand`,
```
Comment (rianfowler) [inline web/src/components/UploadAirgapBundle.jsx:318]: Should we preserve this call and have this copy updated in the troubleshoot app?

**Review by rianfowler — DISMISSED** [review]
added one comment

File: web/src/components/UploadAirgapBundle.jsx (hunk 1 above)
Comment (miaawong) [reply web/src/components/UploadAirgapBundle.jsx:318]: @rianfowler That might be a good idea. Who would reach out to?

Comment (rianfowler) [reply web/src/components/UploadAirgapBundle.jsx:318]: I would just tag the requester and jason on the ticket. something like "we can hardcode this on the FE but it looks like it's currently getting it from a troubleshoot api."

Comment (banjoh) [reply web/src/components/UploadAirgapBundle.jsx:318]: I think this makes sense since the response will have the correct troubleshoot command.

**Review by banjoh — DISMISSED** [review]

File: web/src/components/UploadAirgapBundle.jsx [hunk 2]
```diff
@@ -27,7 +27,10 @@ class UploadAirgapBundle extends React.Component {
     fileUploading: false,
     registryDetails: {},
     preparingOnlineInstall: false,
-    supportBundleCommand: "kubectl support-bundle --load-cluster-specs",
+    supportBundleCommand: [
```
Comment (banjoh) [inline web/src/components/UploadAirgapBundle.jsx:30]: Nit pick: Can this be DRY'd by using a constant to store this commands array?

File: integration/replicated/tests/kitchen-sink/expected/upstream/manifests/support-bundle.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ spec:
         collectorName: ping-google
         image: flungo/netutils
         command: ["ping"]
-        args: ["www.google.com"]
+        args: [["-w", "1", "www.google.com"]
```
Comment (banjoh) [inline integration/replicated/tests/kitchen-sink/expected/upstream/manifests/support-bundle.yaml:22]: This does't look right

File: pkg/supportbundle/supportbundle.go [hunk 4]
```diff
@@ -129,7 +129,7 @@ func GetBundleCommand(appSlug string) []string {
 
 	command := []string{
 		"curl https://krew.sh/support-bundle | bash",
-		fmt.Sprintf("kubectl support-bundle %s --redactors=%s\n", GetSpecURI(appSlug), redactors),
+		fmt.Sprintf("kubectl support-bundle --load-cluster-specs --redactors=%s\n", redactors),
 	}
```
Comment (banjoh) [inline pkg/supportbundle/supportbundle.go:133]: Redactors are also discovered but the support-bundle spec, but I'm not sure whether KOTS labels them at the moment. I think we can address this in a separate GH issue.

**Review by banjoh — APPROVED** [review]

**Pull request merged**